pavelib/prereqs.py

```diff
@@ -164,6 +164,7 @@ def install_node_prereqs():
     "django-storages",
     "django-oauth2-provider",       # Because now it's called edx-django-oauth2-provider.
     "edx-oauth2-provider",          # Because it moved from github to pypi
+    "ccx_keys",                     # Package renamed to edx-ccx-keys and published to PyPI
```

Comment:
If the package name (the installed package, not the downloadable kit) is unchanged, then we don't need an uninstall step, and therefore don't need a clean build?